src/Factory/CustomTransportFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Consolidation\SiteProcess\Factory;
+
+use Consolidation\SiteAlias\SiteAliasInterface;
+use Consolidation\SiteProcess\Transport\CustomTransport;
+use Consolidation\Config\ConfigInterface;
+
+/**
+ * CustomTransportFactory will create an CustomTransport for applicable site aliases.
+ */
+class CustomTransportFactory implements TransportFactoryInterface
+{
+    /**
+     * @inheritdoc
+     */
+    public function check(SiteAliasInterface $siteAlias)
+    {
+        return $siteAlias->has('command');
```

Comment:
This seems too generic

Reply:
@greg-1-anderson, I agree. I hesitated to put it in but on thinking more, it started making sense as the functionality is really that generic. That said, I am open to alternatives.